community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -403,6 +403,7 @@ public static void main( String[] incomingArguments, boolean defaultSettingsSuit
     private static void printOverview( File storeDir, Collection<Option<File[]>> nodesFiles,
             Collection<Option<File[]>> relationshipsFiles )
     {
+        System.out.println( "Neo4j version: " + Version.getKernel().getVersion() );
```

Comment:
Is there a `Version.getKernel().getReleaseVersion()` in 2.2? If there is it might be better to use it because it gives the version without including the git hash. If there isn't, then I suggest looking for it when forward merging and using it as soon as it appears. Don't think there is any need to print out the git hash in this output.
See Mattias change at https://github.com/neo4j/neo4j/blame/2.3/community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java#L522 for something similar (but there it was included in a URI, which meant the git hash actually broke things).